apps/demo/src/app/cart/components/cart-sidebar/cart-sidebar.component.ts

```diff
@@ -1,30 +1,27 @@
 import { Component, Input, OnInit } from '@angular/core';
-import { Store, select } from '@ngrx/store';
-import { Observable } from 'rxjs';
 
 import { DaffCart } from '@daffodil/cart';
+import { Observable } from 'rxjs';
+import { Store, select } from '@ngrx/store';
 
 import * as fromCart from '../../selectors/cart-selector';
 
 @Component({
-  selector: 'demo-cart-wrapper',
-  templateUrl: './cart-wrapper.component.html',
-  styleUrls: ['./cart-wrapper.component.scss']
+  selector: 'demo-cart-sidebar',
+  templateUrl: './cart-sidebar.component.html',
+  styleUrls: ['./cart-sidebar.component.scss']
 })
-export class CartWrapperComponent implements OnInit{
-  
-  @Input() cart: DaffCart;
+export class CartSidebarComponent implements OnInit {
 
   isCartEmpty$ : Observable<boolean>;
```

Comment:
This should be passed as a boolean input, so this component doesn't need to touch the store.